<launch>
	<node pkg="nodelet" type="nodelet" name="manager" args="manager" output="screen"/>
	<node pkg="nodelet" type="nodelet" name="cam" args="load usb_cam/usb_cam_nodelet manager">
		<param name="video_device" value="/dev/video1" />
	    <param name="camera_info_url" type="string" value="file://$(find usb_cam)/cfg/head_camera.yaml"/>
	    <param name="image_width" value="320" />
	    <param name="image_height" value="240" />
	    <param name="camera_frame_id" value="usb_cam" />
	    <param name="framerate" value="5" />
	    <param name="pixel_format" value="yuyv" />
  	</node>
	<node pkg="nodelet" type="nodelet" name="ar_pose" args="load ar_pose/ARMultiNodelet manager">
		<param name="marker_pattern_list" type="string" value="$(find usb_cam)/patterns/column_markers"/>
		<param name="threshold" type="int" value="80"/>
		<param name="publish_tf" type="bool" value="true"/> 
		<remap to="/cam/camera_info" from="/usb_cam/camera_info" />
		<remap to="/cam/image_raw" from="/usb_cam/image_raw" />
	</node>

</launch>
